client/src/styles.css

```diff
@@ -0,0 +1,31 @@
+@import url('https://driftcss.netlify.app/css/drift.css');
+
+body {
+   background: hsl(180, 5%, 96%);
+   position: relative;
+}
+
+.container {
+   max-width: 1400px;
```

Comment:
avoid using px as a measure as much as possible......it sort of reduces the responsiveness of the element

Reply:
I don't know the other option for setting the screen size I might have to look into it.
Thanks for pointing it out!